### ISSUE TITLE
How do landings/ILS work?

### ISSUE THREAD

**Issue by joallard** [zlsa/atc#171]
Since I've been playing from the beginning, I've never understood clearly how ILS worked. Sometimes I will activate it, and it won't find the lock and I will have no idea why. May we have a primer or maybe some explanation about how it works?

**Comment by zlsa** [comment]
For ILS to lock, you must be within 10 degrees (I think) and within the distance of the faint line extending from the runway.

**Comment by joallard** [comment]
> within the distance 
> So is there a minimal distance? Or is this a maximum?

Are those bugs?
<img width="193" alt="screen shot 2015-09-08 at 13 42 42" src="https://cloud.githubusercontent.com/assets/949482/9742730/0a0eee18-5630-11e5-943a-b1f0bc5862cf.png">
<img width="247" alt="screen shot 2015-09-08 at 13 45 27" src="https://cloud.githubusercontent.com/assets/949482/9742736/0f349cbc-5630-11e5-87f0-d6594ea2cc25.png">

Sometimes the plane will trace a path to go on the faint line smoothly, sometimes it will just ignore and run amok.

**Comment by glangford** [comment]
Try to be at 6000' elevation or so. In the second picture , KLC806 is at 8000 and you won't get ILS capture there.

**Comment by glangford** [comment]
Picture a radio beacon pointed up at 3 degrees into the sky; that creates the glideslope. The end of the faint line is the maximum distance you can acquire the ILS. At that point or closer, if you are within 10 degrees of the extended runway line and pointed in that general direction AND not too high, you will lock.

Try KSFO for example at 6000' feet at MEHTA, or 4000' at DUMBA.

**Comment by joallard** [comment]
Okay. Does that mean an aircraft at 8000 will be able to lock but just farther? I'll try that next time, though not today because the game gets addicitive ;o).

I think what I'll try to do is test some cases and bounce them here, and document the wiki based on what's said here.

**Comment by glangford** [comment]
> Okay. Does that mean an aircraft at 8000 will be able to lock but just farther?

No. 8000' is too high, and the ILS signal (for game purposes) won't lock - the faint line extending from the runway tells you the maximum distance you have. If you are at 6000' or 7000' at the end of that line, you are in good shape.

In the real world of course, the ILS radio signal only travels so far and the game reflects that.

**Comment by joallard** [comment]
Okay. What about my first screenshot? Why wouldn't the aircraft lock in that case? And as I understand it there is no minimum distance?

**Comment by glangford** [comment]
> Okay. What about my first screenshot? Why wouldn't the aircraft lock in that case?

I can't tell from the picture.

> And as I understand it there is no minimum distance?

Not that I am aware of, but from a practical standpoint of course you have to keep descending the closer you get to the runway. 

The best recommendation I can make is to get the actual approach charts for the airports you are interested in. Here is KSFO 28L for example; be around DIVEC at 7000 pointing in the general direction of the runway and you are good to go:

![sfo 28l](https://cloud.githubusercontent.com/assets/3035114/9746359/882263e2-5645-11e5-8a96-8fba66a8585e.png)

**Comment by tedrek** [comment]
The issue with both pictures is that the aircraft is too high for how close it is to the runway.  The ILS range at EHAM is reduced on half of the runways from the default of 40km to 28km, therefore to get an intercept you need the aircraft to be at about 5,000 feet or less at the outer end of 36c and 36r.

My rule of thumb has been to vector for an intercept at 4 to 6 thousand in the outer third of the center line, and 2-3 thousand at the inner circle on the radar.  This gives some vertical separation when lining up multiple aircraft on parallel runways.

Although there is no minimum distance specifically, you are unable to clear an aircraft lower than 1000 feet and therefore will be too high to intercept closer than about 2 miles.  At that distance the width of the signal would also make it hard for an aircraft to turn fast enough to intercept so it's not a practical limitation.

**Comment by nielsvdweide** [comment]
Tedrek is correct. For both pictures the aircraft is too close to the runway to lose it's current height and make a stable approach to landing. Keep in mind an airplane also needs to slow down the last bit of the approach. I normally clear aircraft to 4000 or 5000 feet at the end of the extended centreline and that always works! 

The thing with ILS is that it gets more sensitive the closer you get to the station (on the ground abeam the runway). The normal procedure is to let an airplane intercept the glideslope from below and let them intercept the localizer first. (also a law requirement, you can not descend on a glideslope if you are outside half scale deflection on the localizer). Hope this explains a bit more!

**Comment by glangford** [comment]
Good stuff @tedrek and @nielsvdweide - I didn't recognize that the picture was from EHAM.

For what it's worth, here is the chart for EHAM 36R. According to that, being at 3000' for fix NV is the approach, which is consistent with the recommendation at 4000' or 5000' at the end of the centreline. (Maybe we should embed this in small print at the end of the runways?)

![eham 36r](https://cloud.githubusercontent.com/assets/3035114/9747712/69196d4e-564d-11e5-97e0-fb12021a15d7.png)

**Comment by nielsvdweide** [comment]
I don't know where you got that chart for EHAM 36R from, but i will upload an approach plate used by pilots for that same runway! I think it is okay to use your graph and to see at what distance you should be at what altitude but that's not the way ATC does it. Normally atc lets you intercept the ILS from below, let's say from 2000 feet, as you can see in this picture below. You can see that at 6.2 ABA (distance from the treshold) you are at 2000 feet! And that is the same as in your picture! let me know what you think @glangford @tedrek 

<img width="505" alt="schermafbeelding 2015-09-08 om 23 40 35" src="https://cloud.githubusercontent.com/assets/14002770/9748228/aba8b198-5683-11e5-9c2a-eabb58ab3827.png">

**Comment by glangford** [comment]
Ah, that's a much nicer chart. It's good that the information is consistent!  :-) I am not in Europe so I just downloaded the first good looking item I could find from a search engine, which was at ais-netherlands.nl.

**Comment by nielsvdweide** [comment]
Ais-netherlands.nl is a generic website with all the information regarding dutch airspace and airports. I don't think they include all the approach plates and charts for airports. This plate is from jepessen, where most airliners fly with. They use these kind of plates to prepare for the approach. I will include the full picture for you, so you can take a look at it. Note that the procedure on the plate is just a procedure. This doesn't include radar vectors.

<img width="522" alt="schermafbeelding 2015-09-08 om 23 38 35" src="https://cloud.githubusercontent.com/assets/14002770/9748925/ac306746-5688-11e5-940e-a6d89e3099ac.png">

**Comment by glangford** [comment]
Excellent, thank you @nielsvdweide !

**Comment by nielsvdweide** [comment]
No problem! I have some ideas in my mind I will upload tomorrow when I have some spare time. I'm not known to github or scripting, I leave that to other guys! I like this project and maybe I can help improving it :) anyway, have a good nights rest!

**Comment by glangford** [comment]
@joallard can this issue be closed now?

**Comment by joallard** [comment]
I don't mind as to the status of the issue, but I must admit I haven't reached the point where I can ascertain whether or not locking a landing in Atc will work.

**Comment by Maverick283** [comment]
@joallard I don't quite know what gives you trouble, but here is a short list of things to have done in order for aircraft to lock on the ILS:
1. Have the aircraft fly within 30° towards the runway, meaning if it is runway 12, the aircraft must fly within a heading of 110 and 130. Best way to accomplish this by having it fly fixes along the extended centerline (if there are any)...
2. (generally speaking and not always correct!) Have the aircraft fly in between 2000ft and 3000ft above the runway when it is at the end of the little drawn extended centerline. 
3. The plane will not lock unless it is closer to the airport then the beginning of the extended centerline. 

In other words, try getting the aircraft to fly on the centerline at 2000ft above the airport, and then lock it on the correct ILS. 
Hope this helps.
